source/_patterns/00-atoms/_text.twig

```diff
@@ -1,4 +1,8 @@
-{%- if nodes is iterable -%}
+{%- if nodes is not defined -%}
+
+  {# Do nothing #}
```

Comment:
Not sure what is caused this. Only thing I can think of is that Pattern Lab is trying to render these templates with no data...

Reply:
Confirmed in de62246343b28d5148ab789fd68e5163c33cbdb2.